### DISCUSSION TITLE
Typesense for armv7

### DISCUSSION THREAD

**Discussion by titro** [immich-app/immich#2111] [General]
With the new typesense functionality i am not able to run immich in my rpi with armv7 anymore.
Any hints for getting typesense running with v7? I already tried to build it in my own but the dev typesense Image does not Support armv7...

**Comment by alextran1502** [comment]
I think this is the requirement at the moment. We will work on making Typesense as an optional service in the future. Right now you can spin up Typesense on a different machine that support the architecture and point the Immich stack to it
